### ISSUE TITLE
Review IT2 Items 

### ISSUE THREAD

**Issue by lortrent** [lortrent/projects#584]
#Privacy of Information and Protection:
**Your opinion is your personal information. Please do not include any information which identifies you or others in your response.**

#Description:
1) ~Decide if we need the ability for anyone on the route map to send back for corrections or expect the manager to make corrections before sending on or approving in Job Req approval route map..
Should be tested.~  can decline to send back and approve to move forward again.~

2) ~Workflow for Disability - Return to Work in EC. Rizing/CRD Team~
3) Workflow -Return to Work from Gradual Return to Work in EC. Rizing/CRD Team
4) ~Propagate from job code to position in EC. Includes: Pay scale structure ...~  **COMPLETED in IT1.**
5)  ~Reorder Manager Tasks in Onboarding.  They appear to sort alphabetically. We want them to appear in a chronological order. CRD Team~  **CANNOT BE DONE.**
6)  ~Explain how to hire employees who did not get migrated through recruitment and new hire template in EC. Rizing to create a process.~ **Test in IT2**
7) ~Removed Desired Salary from the Std questions on job application in Recruitment.  **@cegede  to send to PAT in the Defect register for IT2**~
8) ~Test switching from an internal to external posting in Recruitment. **CRD Team**~
9) ~Test adding additional comments to the job posting footer by the HR Advisor (Recruiter).  This is required for Open until filled.  Review dates are: Feb 19, 2024,~  **CRD Team**  **DONE.**
10) ~Workflow for updates and approval of Job profile changes in EC.  Managers update job profile approved by HR OM Manager. Rizing/CRD Team~  **DONE. Test in IT2**
11) ~Do we need a custom portlet for job code/profile or just training from employee profile, Actions, Job Profiles search.  DONT NEED a CUSTOM PORTLET.~  **TRAINING TO SEE JOB PROFILES.**
12)  ~Discuss with DocuSign, Payroll logging into the see all compliance forms for new hires or the best way to do that.  SAP Technical and @lortrent  **CRD Team to meet with Payroll.~ **Payroll with have access to manage Documents in SF.**
**13a) When a position is updated workflow in EC.  Change reason is mandatory. 
~and a notification should go to the manager/supervisor. Which managers? Rizing/CRD Team **~
**13b) Global setting "To Be Hired" =Yes on Position only if there is a current FTE value in EC. Rizing.**
14) ~Only an exempt manager can initiate a job requisition.  Job level != Individual Contributor or Supervisor in RBP in EC. Rizing~ DONE. EXEMP MANAGER Role in RBP  Supevisors have been added
15) ~Add Pay Calendar  and business rule for job changes in EC. A job change cannot take effect in the middle of a pay period and will be set to the beginning of the next pay period. Rizing/CRD Team~  DEFECT RECORDED
16) ~On position we removed job relationships so we cannot have a dotted line to admin clerks.  Decision must be final and recorded in CHG Mgmt tracker. CRD Team~
17) ~Decide where the board org unit lives on the org chart. Determine updates required in ECC.,~ they are employees so same replication required. Hide board from everyone but HR.
18) ~Business Phone number will be entered in AD and update the employee profile in SF. SOP required and AD technical work. CRD Team.~
21) ~Email notification benefits optional coverage xx days after start date. Rizing/CRD Team NOT DONE~
22) ~Demonstrate/test job share in EC.  Apply global control attribute=Yes so FTE rule is applied.  No job share on supervisory roles. What does it look like on the org chart?~ Rizing/CRD Team. - **Test IT2**
23) Review Chief Position on ECC./EC. Rizing/CRD Team** @lortrent  - NEW TICKET
24) ~Test Payroll in ECC SBX3 to ensure new config items and software have not impacted Payroll.  Must be completed before end of IT2 so we can move to ECC Dev.~
25) ~On name change workflow, force an attachment and provide a message in EC. Rizing/CRD Team.~  **Test IT2**
26) ~My Profile, remove Badges and Tags? Rizing~ DONE.
27) ~Enable Outlook integration. No room upload into SF.  Manual step to add a room and Teams link in Outlook. ~ DONE.
28) ~Hours of Work pick list on the job req may not work for some jobs.  This field should not be mandatory.  Recruiter will have to add to the offer letter manually~ DONE.
29) Build remaining offer letter templates. Rizing to teach CRD Team. CRD Team to complete. CRD Team/Rizing
30) ~Interview Central - will only be used for a thumbs up/down  and a final score on candidates and to upload notes or take notes~.  DONE.
31) ~How can we identify Senior Mgrs, GMs, managers in SF?  Can we use Pay scale? Do we need to add anther field on employee?~ Can only use job level on position.
32)~ RBP-Allow all employees to access the position org chart. Rizing TEST IT2~
33) ~Deferred Salary program workflow for going on LOA and returning from leave.  Signing up for deferred salary program will be managed outside of SuccessFactors.~  TEST IT2.
34) ~Create all end-to-end tests to be used by HR Working group. CRD Team~
35) ~Create and associated all replication tests to end to end tests. CRD Team~
36) Enable global transition settings. # days or months. Rizing. **Set to 180 Days.**
37) Test term positions. CRD Team
38) Develop report for: Cost Centre Changes on Position - Finance
39) Develop report for: HR Manager - Summary of Portfolio Changes.
40) Develop report for: Empty Positions -to delimit them.
41) ECC Report for data migration. Changes made after Jan 1, 2023 that are effective from Jan 1 2023.  Needed for cut-over.
42) Develop report: For term positions coming due or past the end date to follow-up with hiring managers and update term end dates as required.
43) Develop reports by CBA classification with pay structure. Not sure what is required here.
44) Develop reports to check and maintain data stds and consistency.

#Outcome:
- [x]  All completed except reports.  Reports will be added to the roadmap.
- [ ] 
- [ ] 

#To Complete this ticket:
- [ ]  
- [ ] 
- [ ]
